hooks/boot/05-environment.rb

```diff
@@ -0,0 +1,5 @@
+%w[http_proxy https_proxy ssl_cert_file HTTP_PROXY HTTPS_PROXY SSL_CERT_FILE].each do |variable|
+  if ENV.delete(variable)
+    $stderr.puts "Unsetting environment variable #{variable}"
```

Comment:
I would think for standardization we should use the logger here to output this. And I'd consider doing it as a warning rather than a full on error.

Reply:
Yes, this was just the quickest way that I knew to show the concept.